### DISCUSSION TITLE
Could someone explain to me, what this means?

### DISCUSSION THREAD

**Discussion by Mikael-Kolehmainen** [anuraghazra/github-readme-stats#2303] [Q&A] [answered]
As the title says, how do I fix this? It showed up on my github profile after I had been using it for a while.

![image](https://user-images.githubusercontent.com/58134156/204343428-72316109-2fe2-47fe-8dad-50e1aa19dc5f.png)

Thanks!

**Comment by bylickilabs** [comment]
@Mikael-Kolehmainen 
what is that? I have no problems on my own Github Profile

**Comment by Qubik65536** [comment]
Have you deployed your own API? If so, did you followed the guide at the end of README?

------

Edit: And it looks fine now as I visit your profile

**Comment by rickstaa** [comment, marked as answer]
@Mikael-Kolehmainen Thanks for your question. This error is thrown when our public Verel instance is rate limited (see https://github.com/anuraghazra/github-readme-stats/issues/1471). As explained by @Cubik65536, the best way to prevent this is to deploy your own Vercel instance (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance).

**Reply by Mikael-Kolehmainen** [reply]
Ah, okay. Thank you!
